client/src/components/Crypto/Crypto.js

```diff
@@ -84,45 +108,44 @@ class Crypto extends Component {
                 <h1>ABC</h1>
                 {this.state.allComments.map(parent => 
                     <div className="parentComment" id={"parent"+parent.id} data-id={parent.id} data-parent={true}>
```

Comment:
```suggestion
                    <div className="parentComment" id={"parent"+parent.id} key={"parent"+parent.id} data-id={parent.id} data-parent={true}>
```

I have found that whenever we are creating lists or using a .map function to render similar divs we should include the a unique key attribute when using React. 

"Keys help React identify which items have changed, are added, or are removed. Keys should be given to the elements inside the array to give the elements a stable identity"

https://reactjs.org/docs/lists-and-keys.html